e2e_test/source_inline/kafka/avro/glue.slt

```diff
@@ -14,6 +14,8 @@ EOF
 statement ok
 ALTER SYSTEM SET license_key TO '';
 
+sleep 1s
```

Comment:
There are also many other occurrences which may explode in the coming days, I suppose. 🤣

It's indeed possible to return the `ALTER SYSTEM` statement only after the notification has been broadcast to all nodes in the cluster, but the process of receiving and handling the notification remains asynchronous. So I'm afraid we are not able to completely eliminate the flakiness.

Reply:
> handling the notification remains asynchronous

Is it possible we make it a synchronous process on meta by altering the parameter before repling to fe?

Reply:
It is synchronous on meta node, but the following check is performed on the frontend IIUC.